.travis.yml

```diff
@@ -40,9 +47,13 @@ script:
   - make install.deps
   - make containerd
   - sudo PATH=$PATH GOPATH=$GOPATH make install-containerd
-  - make test
-  - make test-integration
-  - make test-cri
+  - if [ "$HOSTTYPE" == "s390x" ];then
```

Comment:
would prefer to open an issue to add support for adding additional host types, such as using multi arch images.. than to remove the integration and cri tests...

Reply:
@mikebrow We have **raised the issue** for the integration and validation test failures seen on s390x due to the images being used in the test are not available for s390x - https://github.com/containerd/cri/issues/1366.